### PR TITLE
i18n(es): Update remaining files

### PR DESCRIPTION
**Pull request by agustinmulet**
#### What kind of changes does this PR include?

- New or updated content

#### Description

Puts the remaining files up to date, in spanish:

- guides/typescript.md
- install/auto.md
- install/manual.md
- integrations/integrations.md
- reference/api-reference.md
- reference/configuration-reference.md

### CODE REVIEW

**Review by kevinzunigacuellar — COMMENTED** [review]
amazing work! I left a few comments 🚀

File: src/pages/es/guides/deploy.md
Comment (agustinmulet) [reply src/pages/es/guides/deploy.md:82]: I'll add this changes but this file needs a complete review, it has some serious redaction issues

File: src/pages/es/guides/typescript.md
Comment (fredoist) [inline src/pages/es/guides/typescript.md:195]:
```suggestion
**Solución**: Aún no hay una forma confiable de solucionar este error, sin embargo, algunas soluciones y discusiones pueden encontrarse en [esta discusión en GitHub](https://github.com/johnsoncodehk/volar/discussions/592).
```

File: src/pages/es/reference/api-reference.md
Comment (fredoist) [inline src/pages/es/reference/api-reference.md:42]:
```suggestion
- No necesitas cargar cada archivo de inmediato. `import.meta.glob()` puede devolver funciones que importan el contenido del archivo, en vez de devolver el contenido en sí.
```

**Review by fredoist — CHANGES_REQUESTED** [review]
Una ultima corrección.

**Comment by kevinzunigacuellar** [comment]
✅ ready to merge

**Review by fredoist — APPROVED** [review]

**Comment by Yan-Thomas** [comment]
Good job everyone! I'll merge this PR once you have resolved the merge conflicts and got a final LGTM so I know everything is in place, thank you!

**Comment by fredoist** [comment]
Great work @agustinmulet 🤝

LGTM ✅

**Comment by Yan-Thomas** [comment]
Thanks again, everyone! Really an amazing job y'all are doing here 💜

**Pull request merged**
